gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Mon Mar 23 14:55:28 PDT 2015
+#Fri Mar 27 16:28:33 PDT 2020
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.3-all.zip
```

Comment:
This change is intentional for this PR, right? If so, it's ok to keep it; just double checking since it isn't quite related.

Reply:
It is auto updated by the build tool. I think I can revert this to keep it separated.

Reply:
It's pretty minor. It's ok to leave it in this PR.